lib/smartcar/vehicle.rb

```diff
@@ -181,7 +183,8 @@ def initialize(token:, id:, options: {})
                         when :delete
                           delete(item[:path].call(id))
                         else
-                          fetch(path: item[:path].call(id))
+                          query_params = @flags.nil? ? {} : { flags: @flags }
+                          fetch(path: item[:path].call(id), query_params: query_params)
                         end
         build_aliases(build_response(body, headers), item[:aliases])
       end
```

Comment:
Also need to add the flags into the methods listed below:
- permissions
- subscribe
- unsubscribe
- batch
- request

There might be some fancy way that you can set the flags in one place and have them be picked up by all of the methods. @s-ashwinkumar will know if there is

Reply:
With a little bit of modification to the [define_method block](https://github.com/smartcar/ruby-sdk/blob/master/lib/smartcar/vehicle.rb#L172-L188) we can add this into the[ config that  is used to generate those methods](https://github.com/smartcar/ruby-sdk/blob/master/lib/smartcar/vehicle.rb#L19-L71)